### ISSUE TITLE
[BJ] Reporting grabs as seeds, and seeds as leechers, some movies are not found by radarr

### ISSUE THREAD

**Issue by Nottt** [Jackett/Jackett#2092]
Jackett version:  1.0.0.0
Mono version (if not using Windows):  Mono JIT compiler version 5.4.0.201 (tarball Fri Oct 13 21:51:45 UTC 2017)

This is my logs. I also searched by a movie in radarr that didn't show up results using this tracker, and then searched it manually and it appeared. There are no error logs in radarr relating indexers only this :

```
17-11-8 08:05:46.1|Info|NzbSearchService|Searching 2 indexers for [Pete's Dragon]
17-11-8 08:05:46.6|Info|DownloadDecisionMaker|Processing 14 releases
```

I can also get a invite if necessary. 

-------------------------------
[jackett-log.txt](https://github.com/Jackett/Jackett/files/1453029/jackett-log.txt)

**Comment by Guizzoni** [comment]
I'll work on a fix for the "Grabs/Seeders/Leechers" issue.

**Comment by kaso17** [comment]
fixed via 804dc12a4770119fa69b48e209cff988964402b2
